core/os/device/device.proto

```diff
@@ -269,3 +271,18 @@ message VulkanPhysicalDevice {
   // deviceName is a null-terminated string containing the name of the device.
   string device_name = 5;
 }
+
+// DeviceValidationCache contains the list of validated devices.
+// The cached information is stored into a file whenever a new device is
+// successfully validated.
+message DeviceValidationCache {
+  repeated DeviceValidationCacheEntry entries = 1;
+}
+
+// DeviceValidationCacheEntry describes the properties of the device that can
+// change validation results if updated.
+message DeviceValidationCacheEntry {
+  OS os = 1;
+  repeated VulkanPhysicalDevice physical_devices = 2;
```

Comment:
let's store the driver_version instead. And make sure to put a comment saying this is per `vkPhysicalDevice`

Reply:
Done.